home/factories.py

```diff
@@ -121,16 +121,27 @@ class FooterIndexPageFactory(PageFactory):
     class Meta:
         model = FooterIndexPage
 
+
 class BannerIndexPageFactory(PageFactory):
     title = 'Banners'
 
     class Meta:
         model = BannerIndexPage
 
-class BannerFactory(PageFactory):
 
+class BannerFactory(PageFactory):
     title = factory.Sequence(lambda n: f'banner{n}')
     banner_image = factory.SubFactory(ImageFactory)
+
     class Meta:
         model = BannerPage
 
+
+class FlatMenuFactory(DjangoModelFactory):
+    class Meta:
+        model = FlatMenu
+
+
+class IogtFlatMenuItemFactory(DjangoModelFactory):
```

Comment:
I wonder whether this is really helping us to create flat menu items. What would happen if we did not use a factory?